app/src/main/java/com/hedvig/app/feature/home/ui/claimstatus/data/ClaimTypeData.kt

```diff
@@ -0,0 +1,28 @@
+package com.hedvig.app.feature.home.ui.claimstatus.data
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.res.stringResource
+import com.hedvig.android.owldroid.graphql.HomeQuery
+import com.hedvig.android.owldroid.type.ClaimStatus
+import com.hedvig.app.R
+
+@JvmInline
+value class ClaimTypeData(val text: String) {
+    companion object {
+        @Composable
+        fun fromClaimStatus(claimStatus: HomeQuery.ClaimStatus): ClaimTypeData {
+            return ClaimTypeData(
+                when {
+                    // TODO get the type from backend? It exists in Hope already, but not on GraphQL 
+                    // claimStatus.perilType != null -> claimStatus.perilType.toProperReadableString
```

Comment:
Hope does contain information on "Claim Type" when looking at a specific claim, something that doesn't seem to be available in the GraphQL query.
Need to edit giraffe? Not sure, but need to add this so that not all the claims show "(New) Insurance case"

Reply:
If it's not in the API, we should discuss it with the implementing backend team (MX) 🙂